main/core/src/mill/eval/Evaluator.scala

```diff
@@ -94,6 +94,13 @@ case class Evaluator(
       testReporter: TestReporter = DummyTestReporter
   ): Evaluator.Results = {
     val (sortedGroups, transitive) = Evaluator.plan(rootModule, goals)
+    val terminalNames = sortedGroups
+      .keys()
+      .map{
+        case Left(x) => x
+        case Right(x) => (System.identityHashCode(x), x.task.ctx.lineNum, x.segments.render)
+      }
+      .toList
```

Comment:
What happens here and where is `terminalNames` used?

Reply:
It's not used, this is leftover debugging logic. Will remove